Update packages for 8.1.0

randomir opened the pull request.

JoelPasvolsky commented:
Some easy fixes here: 

- [line 72](https://github.com/dwavesystems/dwave-ocean-sdk/blob/master/docs/examples/pp_greedy.rst): `from greedy import SteepestDescentSolver`
- [line 75](https://github.com/dwavesystems/dimod/blob/9b17267e2db2db0f0687e63741db05ed6ca6df14/dimod/reference/composites/tracking.py#L75): `        OrderedDict([('h', {'a': -1}), ('J', {('a', 'b'): 1}), ('num_reads', 5)])`
- [line 85](https://github.com/dwavesystems/dimod/blob/9b17267e2db2db0f0687e63741db05ed6ca6df14/dimod/reference/composites/tracking.py#L85): `OrderedDict([('Q', {('a', 'b'): 1})])`
- [line 16](https://raw.githubusercontent.com/dwavesystems/dwave-system/60a4e4b5c4c42170e2c74ceb8f3231d0a0d7fdc7/docs/reference/warnings.rst): `>>> import greedy`

randomir commented:
Doctests started failing on this PR because, although we dropped `dwave-{greedy,neal,tabu}` in https://github.com/dwavesystems/dwave-ocean-sdk/pull/312, they were still being installed by `dwave-hybrid`. We switched to `dwave-samplers` completely in `dwave-hybrid==0.6.13` (latest version used in this PR).

arcondello commented on .appveyor.yml:
We really ought to just drop appveyor at some point

randomir replied:
Absolutely. Every time I'm surprised to see appveyor conf in the sdk :slightly_smiling_face: 
I can switch to CircleCI in a follow-up PR.

arcondello reviewed (approved):
LGTM

thisac reviewed (approved):
LGTM

randomir commented:
Thanks @JoelPasvolsky for catching that -- there was an issue with `doxygen` install (see 63b47daf447a69a9d5c83f2fc3bd153da356cac4). The output now [looks correct](https://output.circle-artifacts.com/output/job/984f3279-6d09-4812-8ab2-4ddc6e57a76e/artifacts/0/docs/_build/html/docs_dimod/reference/cpp.html). Please feel free to re-review. Thanks!

JoelPasvolsky reviewed (approved):
Thanks!
We should do something about [dwave-gate qir](https://output.circle-artifacts.com/output/job/984f3279-6d09-4812-8ab2-4ddc6e57a76e/artifacts/0/docs/_build/html/docs_gate/reference/qir.html) but that's not here.
